src/Resources/contao/classes/DataContainer.php

```diff
@@ -447,6 +466,45 @@ protected function row($strPalette=null)
   </script>';
 		}
 
+		// Page picker
+		if ($arrData['eval']['dcaPicker'])
```

Comment:
I think it would be nicer to combine the `eval.dcaPicker*` keys into a single `dcaPicker` array like `'eval' => ['dcaPicker' => ['do' => 'files, 'icon' => '']]`.